### PR TITLE
Adjust load flow results

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/components/results/loadflow/load-flow-result-utils.ts [hunk 1]
```diff
@@ -101,8 +89,42 @@ export const loadFlowCurrentViolationsColumnsDefinition = (
...
         },
+        {
+            headerName: intl.formatMessage({
+                id: 'ActualOverload',
+            }),
+            field: 'actualOverload',
+            valueFormatter: (value: ValueFormatterParams) => {
+                return value.data.actualOverload ===
+                    UNDEFINED_ACCEPTABLE_DURATION
+                    ? intl.formatMessage({ id: 'UndefinedOverlaod' })
```
Comment (dbraquart) [inline src/components/results/loadflow/load-flow-result-utils.ts:100]:
```suggestion
                    ? intl.formatMessage({ id: 'UndefinedOverload' })
```

File: src/components/results/loadflow/load-flow-result-utils.ts [hunk 2]
```diff
@@ -101,8 +89,42 @@ export const loadFlowCurrentViolationsColumnsDefinition = (
...
+                    : convertDuration(value.data.actualOverload);
+            },
+        },
+        {
+            headerName: intl.formatMessage({ id: 'upComingOverlaod' }),
```
Comment (dbraquart) [inline src/components/results/loadflow/load-flow-result-utils.ts:105]:
```suggestion
            headerName: intl.formatMessage({ id: 'upComingOverload' }),
```

File: src/components/results/loadflow/load-flow-result-utils.ts [hunk 3]
```diff
@@ -101,8 +89,42 @@ export const loadFlowCurrentViolationsColumnsDefinition = (
...
+            field: 'upComingOverload',
+            valueFormatter: (value: ValueFormatterParams) => {
+                return value.data.upComingOverload ===
+                    UNDEFINED_ACCEPTABLE_DURATION
+                    ? intl.formatMessage({ id: 'UndefinedOverlaod' })
```
Comment (dbraquart) [inline src/components/results/loadflow/load-flow-result-utils.ts:110]:
```suggestion
                    ? intl.formatMessage({ id: 'UndefinedOverload' })
```

File: src/components/results/loadflow/load-flow-result-utils.ts [hunk 4]
```diff
@@ -101,8 +89,42 @@ export const loadFlowCurrentViolationsColumnsDefinition = (
...
+            },
+        },
+        {
+            headerName: intl.formatMessage({ id: 'LimitSide' }),
+            field: 'side',
+        },
     ];
 };
+
+export const formatLimitName = (limiName: string, intl: IntlShape) => {
```
Comment (dbraquart) [inline src/components/results/loadflow/load-flow-result-utils.ts:123]:
```suggestion
export const formatLimitName = (limitName: string, intl: IntlShape) => {
```

File: src/components/results/loadflow/load-flow-result-utils.ts [hunk 5]
```diff
@@ -42,11 +43,8 @@ export const makeData = (
...
         name: overloadedEquipment.subjectId,
         value: overloadedEquipment.value,
-        acceptableDuration:
-            overloadedEquipment.acceptableDuration ===
-            UNDEFINED_ACCEPTABLE_DURATION
-                ? null
-                : overloadedEquipment.acceptableDuration,
+        actualOverload: overloadedEquipment.actualOverload,
+        upComingOverload: overloadedEquipment.upComingOverload,
         limit: overloadedEquipment.limit,
```
[the remark this answers is not in the document]
Comment (souissimai) [reply src/components/results/loadflow/load-flow-result-utils.ts:48]: done

File: src/components/results/loadflow/load-flow-result-utils.ts (hunk 1 above)
Comment (souissimai) [reply src/components/results/loadflow/load-flow-result-utils.ts]: done

File: src/components/results/loadflow/load-flow-result-utils.ts (hunk 3 above)
Comment (souissimai) [reply src/components/results/loadflow/load-flow-result-utils.ts]: done

File: src/components/results/loadflow/load-flow-result-utils.ts (hunk 2 above)
Comment (souissimai) [reply src/components/results/loadflow/load-flow-result-utils.ts]: done

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Pull request merged**